README.md

```diff
@@ -4089,7 +4089,7 @@ In order to achieve greater coverage and encourage more people to contribute to
             </td>
             <td> <!-- Python -->
                 <a href="./CONTRIBUTING.md">
-                    <img align="center" height="25" src="./logos/python.svg" />
+                    <img align="center" height="25" src="./logos/github.svg" />
```

Comment:
For a brief moment based on the PR description, I thought there were existing algorithms that were missing the logos until seeing the commit referring to that.

I sounds like you meant the 10 new algorithms you included in the commit with hash 43363dc52b1886fbcfad88692c489de5342f1117 were missing the logos and the comment was addressing that.

Reply:
Is that correct, @OzPol?

Reply:
Thank you for the contribution, @OzPol. I checked the code, and overall it looks great.
I think once you confirm this question, we should be good to approve/merge this.

Reply:
Hi @ricardocarva,

Thank you for your review and follow-up! Yes, you're correct – the comment regarding the missing logos pertains to the 10 new algorithms added in commit 43363dc. Specifically, I mistakenly copied the Python implementation placeholder from an existing algorithm for some entries (like the Red-Black Tree and Bellman-Ford). I corrected this in a follow-up commit by replacing the incorrect logos with GitHub placeholders to ensure consistency for unimplemented algorithms.

However, before submitting the PR, I noticed a broken link in the README, unrelated to the new additions. I fixed that as well, which initially caused a lint check failure but was resolved afterward.

Please let me know if anything else needs adjustment!
Thanks again for your review and support.